modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc

```diff
@@ -0,0 +1,51 @@
+= Create a BYOC Cluster on Azure
+:description: Use the Redpanda Cloud UI to create a BYOC cluster on Azure.
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+== Prerequisites
+
+* A minimum version of `rpk` v24.1
+* A dedicated Azure subscription: 
++
+** Role: The Azure user must have the _Owner_ role in the subscription.
++
+** Resources: For more information, see the https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types[Microsoft documentation].
++
+*** Microsoft.Compute
+*** Microsoft.ManagedIdentity
+*** Microsoft.Storage
+*** Microsoft.KeyVault
+*** Microsoft.Network
+*** Microsoft.Compute.EncryptionAtHost
```

Comment:
I don't see `*** Microsoft.Compute.EncryptionAtHost` 
![](https://files.slack.com/files-pri/TPMVB7YMC-F07EHMH81SS/screenshot_2024-07-30_at_16.35.21.png)

Reply:
NVM, it's not registered by default.

Reply:
Could we add additional link to https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/preview-features?tabs=azure-portal or mention that EncryptionAtHost is not a provider, but feature within Compute provider?

Reply:
If there is FAQ somewhere maybe 

When there is new subscription in azure user can see the following error
```
2024-07-30T13:41:34.944Z	ERROR	.rpk.managed-byoc	validation/validate.go:72	Checking compute... FAILED
2024-07-30T13:41:34.944Z	ERROR	.rpk.managed-byoc	validation/validate.go:74	Microsoft.Compute resource provider is not registered
2024-07-30T13:42:02.089Z	ERROR	.rpk.managed-byoc	validation/validate.go:72	Checking machine type and cpu availability... FAILED
2024-07-30T13:42:02.089Z	ERROR	.rpk.managed-byoc	validation/validate.go:74	Machine type family quota standardLASv3Family not available for Standard_L8as_v3.
2024-07-30T13:42:02.089Z	ERROR	.rpk.managed-byoc	validation/validate.go:74	Machine type family quota standardDADSv5Family not available for Standard_D2ads_v5.
2024-07-30T13:42:02.089Z	ERROR	.rpk.managed-byoc	validation/validate.go:74	Machine type family quota standardDSv3Family not available for Standard_D2s_v3.
2024-07-30T13:42:03.323Z	INFO	.rpk.managed-byoc	validation/validate.go:68	Checking policies... PASSED
2024-07-30T13:42:04.169Z	ERROR	.rpk.managed-byoc	validation/validate.go:54	An error occurred while checking network usage. GET https://management.azure.com/subscriptions/b820aa73-736b-447c-88d0-4fdf6d42187c/providers/Microsoft.Network/locations/norwayeast/usages
--------------------------------------------------------------------------------
RESPONSE 409: 409 Conflict
ERROR CODE: SubscriptionHasNoUsages
--------------------------------------------------------------------------------
{
  "error": {
    "code": "SubscriptionHasNoUsages",
    "message": "Subscription b820aa73-736b-447c-88d0-4fdf6d42187c has no usages in NRP.",
    "details": []
  }
}
--------------------------------------------------------------------------------
```

As suggested by @c4milo user need to register providers